samples/githubDispatch/command.js

```diff
@@ -0,0 +1,62 @@
+const { Octokit } = require("octokit");
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot trigger-fusebit-pipeline ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+  if (ctx.body.args.length === 0 || ctx.body.args[0] === "help") {
+    return help(ctx);
+  }
+  if (!ctx.configuration.PAT) {
+    await ctx.client.send(
+      "unable to find github PAT in configuration, please set a PAT in configuration with the scope of * on repo and write on action."
```

Comment:
... or just link back to the README.md, e.g. 

> You must specify the PAT configuration property. See {url-to-README.md} for details.

Reply:
Done